docs/source/configuration/commandline.rst

```diff
@@ -94,7 +94,7 @@ Html report related
 [-nopages]
     Just list data as one long list instead of pages.
 [-rails]
-    Use Rails-based naming convention to find relationships between logical foreign keys and primary keys.
+    Use `Rails-based naming convention <https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e#relations-in-models>`_ to find relationships between logical foreign keys and primary keys.
```

Comment:
I wonder if we should find a better example than a gist.

Reply:
You send me the link, I only added to the documentation. Maybe the current implementation should be checked. If the only implemented thing is that a relation is build between
AnyTable.Profiles_ID is linked to Profiles.ID
Then it would be better to write this example